package.json

```diff
@@ -34,9 +32,13 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "js-yaml": "^4.1.0",
-    "mochawesome": "^7.1.3",
     "stylelint": "^15.3.0",
     "stylelint-config-standard": "^31.0.0",
     "stylelint-config-standard-scss": "^7.0.1"
+  },
+  "peerDependencies": {
+    "@badeball/cypress-cucumber-preprocessor": "^17.2.1",
+    "cypress": "12.9.0",
+    "@cypress/webpack-preprocessor": "latest"
```

Comment:
Lets set this to a specific version to avoid future issues

Reply:
Agreed, we should never use `latest` as a version